packages/concerto-core/types/lib/introspect/mapdeclaration.d.ts

```diff
@@ -0,0 +1,96 @@
+export = MapDeclaration;
+/**
+ * MapDeclaration defines an MapType of Key & Value pair.
```

Comment:
Improve description. E.g. 

A MapDeclaration defines a map/dictionary of values indexed by a key.